04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -294,39 +315,420 @@
+    }
+   ],
+   "source": [
+    "qc.mydb_import('anotherresultname','vos://newmags.csv')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "anotherresultname,created:2020-08-28 16:34:03 MST\n",
+      "c4d,created:2020-08-27 14:55:07 MST\n",
+      "cd,created:2020-08-27 15:08:40 MST\n",
+      "mags,created:2020-08-28 15:26:07 MST\n",
+      "newermags,created:2020-08-28 15:49:49 MST\n",
+      "quickresults,created:2020-08-25 14:44:34 MST\n",
+      "quickresults2,created:2020-08-25 14:46:01 MST\n",
+      "resultname,created:2020-08-28 16:34:00 MST\n",
+      "usno_objects,created:2020-08-25 12:10:18 MST\n",
+      "usno_test,created:2020-08-20 12:21:28 MST\n",
+      " \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(qc.mydb_list(),\"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, for clean-up purposes, let's remove the two tables we just imported into MyDB by using the *mydb_drop* function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'OK'"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc.mydb_drop('resultname')\n",
+    "qc.mydb_drop('anotherresultname')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And let's make sure the two tables were removed from MyDB by using the *mydb_list()* function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "c4d,created:2020-08-27 14:55:07 MST\n",
+      "cd,created:2020-08-27 15:08:40 MST\n",
+      "mags,created:2020-08-28 15:26:07 MST\n",
+      "newermags,created:2020-08-28 15:49:49 MST\n",
+      "quickresults,created:2020-08-25 14:44:34 MST\n",
+      "quickresults2,created:2020-08-25 14:46:01 MST\n",
+      "usno_objects,created:2020-08-25 12:10:18 MST\n",
+      "usno_test,created:2020-08-20 12:21:28 MST\n",
+      " \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(qc.mydb_list(),\"\\n\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a class=\"anchor\" id=\"async\"></a>\n",
+    "# An asynchronous query\n",
+    "\n",
+    "We now want to run a longer query, say, counting the total number of objects in USNO-B1, and need to do it asynchronously. The first thing we do is submit the query as normal but with the *async_* argument indicated - this will return the id of the asynchronous job. All of the previous arguments can also be used to specify where and in what format we want the query results."
```

Comment:
So, this section about async queries and result retrieval needs to be adapted so that it can run in auto-testing mode. I.e. use the method
```
qc.query(sql=query,async_=True,wait=True,poll=5)
```
and also explain or link to the "pedestrian" way with getting jobid, checking status until 'COMPLETED', retrieving results with qc.results(). (You can link to the 101 NB for that)